diffkemp/cli.py

```diff
@@ -188,6 +195,31 @@ def make_argument_parser():
                          help="runs development server instead of production \
                          server")
     view_ap.set_defaults(func=diffkemp.diffkemp.view)
+
+    # "compile-pattern" sub-command
+    compile_pattern_ap = sub_ap.add_parser("compile-pattern",
+                                           help="compile a custom diffkemp \
+                                           pattern into valid form")
+    compile_pattern_ap.add_argument("pattern_file",
+                                    help="path to the custom pattern file")
+    compile_pattern_ap.add_argument("--c-pattern-append", help="option to \
```

Comment:
For example, it may be desired to compile the pattern with different optimization level than the default one. Or to define macro values necessary for compilation (the ```--c-pattern-kernel-path``` option automatizes this for kernel patterns, but there are other projects where the user needs to define these macro values at compile time manually)

Reply:
Makes sense. I'd still rename it, though.